packages/@react-aria/tabs/src/useTabs.ts

```diff
@@ -64,19 +63,12 @@ export function useTabs<T>(props: TabsAriaProps<T>, state: SingleSelectListState
   let tabsId = useId();
   tabsIds.set(state, tabsId);
 
-  let [isFocusWithin, setFocusWithin] = useState(false);
-  let {focusWithinProps} = useFocusWithin({
-    onFocusWithinChange: setFocusWithin
-  });
-  let tabIndex = isFocusWithin ? -1 : 0;
-
   return {
     tabListProps: {
-      ...mergeProps(focusWithinProps, collectionProps),
+      ...collectionProps,
       role: 'tablist',
-      'aria-disabled': isDisabled,
       'aria-label': ariaLabel,
-      tabIndex: isDisabled ? null : tabIndex
+      tabIndex: undefined
```

Comment:
can we just omit this property? or is something else setting it in collectionProps?

Reply:
I think collectionProps supplies a tabIndex.